skelebot/systems/execution/docker.py

```diff
@@ -29,7 +29,10 @@ def build(config):
     # Build Dockerfile, .dockerignore, and Docker Image
     dockerfile.buildDockerfile(config)
     dockerignore.buildDockerignore(config)
-    status = os.system(BUILD_CMD.format(image=config.getImageName()))
+    image = config.getImageName()
+    if config.env:
+        image += "-{env}".format(env=config.env)
```

Comment:
Since the env field is part of the config, we can just add this logic directly to the getImageName function in the Config Class.

Reply:
Done with latest commit